tests/nested/core20/degraded/task.yaml

```diff
@@ -0,0 +1,49 @@
+summary: Transition to recover mode with things missing so we use degraded mode
+
+environment:
+  DEGRADED_JSON: /run/snapd/snap-bootstrap/degraded.json
+
+execute: |
+  #shellcheck source=tests/lib/nested.sh
+  . "$TESTSLIB/nested.sh"
+
+  # wait for the system to be seeded first
+
+  nested_wait_for_snap_command
+  nested_exec "sudo snap wait system seed.loaded"
+
+  echo "Install jq in the host environment"
+  snap install jq
+
+  echo "Move the run key for ubuntu-save out of the way so we use the fallback key to unlock ubuntu-save"
+  nested_exec "sudo mv /run/mnt/data/system-data/var/lib/snapd/device/fde/ubuntu-save.key /run/mnt/data/system-data/var/lib/snapd/device/fde/ubuntu-save.key.bk"
```

Comment:
At some point it'd be useful to make this a manual test and check all the variants we support: fully encrypted, unencrypted with ubuntu-save and one without.

Reply:
In the end I've refactored the tests/nested/core20/save spread test in #9640